theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer.tsx

```diff
@@ -66,11 +65,13 @@ export class TraceViewerWidget extends ReactWidget implements StatefulWidget {
 
     private onOutputAdded = (payload: OutputAddedSignalPayload): Promise<void> => this.doHandleOutputAddedSignal(payload);
     private onTraceOverviewOpened = (): Promise<void> => this.doHandleTraceOverviewOpenedSignal();
-    private onExperimentSelected = (experiment: Experiment): void => this.doHandleExperimentSelectedSignal(experiment);
+    private onSelectTraceOverviewOutput = (payload: {traceId: string, outputDescriptor: OutputDescriptor}): Promise<void> => this.doHandleSelectTraceOverviewOutput(payload);
```

Comment:
`onSelectTraceOverviewOutput` should be called `onTraceOverviewOutputSelected`, and `doHandleSelectTraceOverviewOutput` should be called `doHandleTraceOverviewOutputSelected`, right?

Reply:
Yes, I will update the name.